### ISSUE TITLE
"Allow Multiple Selections" not persisting after loading form builder via JSON

### ISSUE THREAD

**Issue by GrumpyCrouton** [kevinchappell/formBuilder#1078]
### Description:

When loading a form in the builder from JSON, select elements are not retaining the "Allow Multiple Selections" option, and it's clearing the selected value.

### Environment Details:

 - formBuilder Version: 3.4.1
 - Browser: Chrome
 - OS: Windows 10 

### Expected Behavior

Select element retains "Allow Multiple Selections" when loaded from JSON.

### Actual Behavior

Select element deos not retain "Allow Multiple Selections" when loaded from JSON.

### Screenshots

Selected options before save:
https://i.ibb.co/gyh8VZp/dropdown-options.png

Stored JSON:
https://i.ibb.co/gyMKcnt/json.png

Selected options after loading JSON in form builder:
https://i.ibb.co/MPvMxyV/after-reload.png

### Form Builder init

```
var options = {
	formData: data, 
	dataType: 'json',
	fields: fields,
	templates: templates,
	typeUserAttrs: typeUserAttrs,
	typeUserDisabledAttrs: {
		dcms_step: ['placeholder', 'className', 'required'],
		dcms_user_data: ['placeholder', 'className', 'required', 'label', 'description']
	},
	disableFields: ['button', 'autocomplete', 'hidden'], 
	disabledActionButtons: ['data', 'save', 'clear'], 
	disabledAttrs: ['access']
};

var formBuilder = $('#fb-editor').formBuilder(options);
```

In regards to `fields`, `templates`, and `typeUserAttrs`, the select option is only mentioned once, and that is in `typeUserAttrs`, it looks like this:

```
var typeUserAttrs = {
	select: {
		hidden: {
			label: 'Hidden',
			options: {
				'false': 'No',
				'true': 'Yes'
			}
		}
	},
}
```

Edit:

I found [this question](https://github.com/kevinchappell/formBuilder/issues/401) which is similar to mine

After reading this, I removed `,"multiple":"false"` from my JSON, and the multiple option is no longer selected when formBuilder is loaded, however, the option that I had selected is still not selected. I also don't want to have to remove `,"multiple":"false"` from my json every time I save the form.

Here is what the form builder does when I remove that line of JSON.
https://i.ibb.co/svv2LGk/multiple-removed-from-json.png

And here is how I save my form:

```
$('.saveform').on('click', function() {
	var json = JSON.parse(formBuilder.actions.getData('json'));
	console.log(json); //multiple option exists in this output
	
	$('body').toast({class: 'yellow', position: 'top center', message: 'Form Saving...'});
	
	http_post("{{ @BASE }}/{{ @module }}/edit?id={{ @GET.id}}", {"form": "{{ @form['json_identifier'] }}", "json": json}, function( data ) {
		$('body').toast({class: 'success', position: 'top center', message: 'Form Saved!'});
		
	});
	
});
```

**Comment by GrumpyCrouton** [comment]
I was able to implement a temporary fix for this by looping through all of the values from `getData()` and removing the `multiple` option from the array. This does not fix the default option selection problem.

```
var json = JSON.parse(formBuilder.actions.getData('json'));

$.each(json, function( index, value ) {
	if(value.type == 'select' && value.multiple === false) {
		delete json[index]['multiple'];
	}
});
```

**Comment by kevinchappell** [comment]
resolved in `v3.5.1`
